.gitignore

```diff
@@ -25,6 +25,8 @@ compile_commands.json
 /bin/wakebox
 /bin/wakebox.*
 /bin/wake-format.*
+/bin/job-cache
+/bin/job-cache.*
```

Comment:
lol, these were added, removed, partially added back in as eviction, removed, and then added back in finally for good I think this time.